README.md

```diff
@@ -22,36 +80,233 @@ If you need to use an illustration that exceeds this value, you can compress it
+                              "en": "Creating a subscription plan",
+                              "es": "Cómo crear un plan de suscripción",
+                              "pt": "Como criar um plano de assinatura"
+                           },
+                           "slug": {
+                              "en": "creating-a-subscription-plan",
+                              "es": "como-crear-un-plan-de-suscripcion",
+                              "pt": "como-criar-um-plano-de-assinatura"
+                           },
+                           "origin": "",
+                           "type": "markdown",
+                           "children": []
+                        }
+                     ]
+                  },
+...
+}
+```
+
+Follow the steps below to add new content in the navigation sidebar:
+
+1. Open a branch in the [helpcenter](https://github.com/vtexdocs/helpcenter) repository.
+
+   > ⚠️ **Before you start adding commits, read the [helpcenter repository's readme](https://github.com/vtexdocs/helpcenter/blob/main/README.md) file!** Commits must be done in a certain format for your PR to be approved.
+
+2. In the [navigation.json](https://github.com/vtexdocs/helpcenter/blob/main/public/navigation.json) file, locate where you want the new reference for the article to appear.
+3. Copy and paste the structure below into **navigation.json**, replacing the names of the articles (three versions) within the `name` object, and the respective slugs in the `slug` object.
+
+   ```jsx
+      {
+         "name": {
+            "en": "Creating a subscription plan",
+            "es": "Cómo crear un plan de suscripción",
+            "pt": "Como criar um plano de assinatura"
+         },
+            "slug": {
+               "en": "creating-a-subscription-plan",
+               "es": "como-crear-un-plan-de-suscripcion",
+               "pt": "como-criar-um-plano-de-assinatura"
+         },
+            "origin": "",
+            "type": "markdown",
+            "children": []
+      }
+   ```
+4. Open a PR on GitHub.
+
+   > ℹ️ By opening a PR, a bot will present a preview for you to test the navigation. With each commit, the preview will be updated.
+
+5. Test your navigation through the preview.
+6. Send the PR link in the `#helpcenter-portal-pr` Slack channel to be approved.
+7. Once PR is approved, apply the merge to update the navigation sidebar.
+
+    > ℹ️ For it to appear in the Help Center Portal, our Tech Writing team leaders must run the portal's build.
+
+
+
+### Updating a published article
+
+To update the content of an article that is already published on the Help Center Portal, follow the steps below:
+
+1. Open a branch in the [help-portal-content](https://github.com/vtexdocs/help-center-content) repository.
+2. Access and update the **PT version** of the desired file in the repository.
+3. Access and update the files in the EN and ES versions with the same information (in Portuguese) inserted in the PT file.
```

Comment:
how about the localization flow for updates? why would you edit en and es directly before step 4 (localization)?

Reply:
We will not describe the internal flow of Localization here.
However, once the content of the PT version is added to the EN and ES versions (Step 3), Localization will perform the translation procedure and replace the translated versions (in the same entered location).
We will explain this in more detail to TWs (presentation or video) once the flow is implemented.